Makefile

```diff
@@ -25,6 +25,7 @@ requirement: roms
 
 roms: $(ROMS)
```

Comment:
If we use a makefile, couldn't we use it as our main interface to build, run, test and get dependencies on top of Cargo?